ts/packages/node/src/stores/OpenFeature.test.ts

```diff
@@ -0,0 +1,97 @@
+import { InMemoryProvider, type EvaluationContext } from '@openfeature/server-sdk';
+import { ConfigSet, ConfigSchema, EvalCommand, ExportCommand, type EvalCommandReturn } from '@configu/ts';
```

Comment:
why testing a store require Commands? this is not e2e tests those are unit tests, you hsould use only the class itself with the get and the set method. please insert a fix
@RonConfigu @RichardAkman